lib/boom_notifier/mail_notifier/templates/email_body.text.eex

```diff
@@ -4,15 +4,24 @@
   <% end %>
   <%= if error.request do %>
     Request Information:
-    URL: <%= error.request.url %>
-    Path: <%= error.request.path %>
-    Method: <%= error.request.method %>
-    Port: <%= error.request.port %>
-    Scheme: <%= error.request.scheme %>
-    Query String: <%= error.request.query_string %>
-    Client IP: <%= error.request.client_ip %>
+      URL: <%= error.request.url %>
+      Path: <%= error.request.path %>
+      Method: <%= error.request.method %>
+      Port: <%= error.request.port %>
+      Scheme: <%= error.request.scheme %>
+      Query String: <%= error.request.query_string %>
+      Client IP: <%= error.request.client_ip %>
+  <% end %>
+  Occurred on: <%= error.timestamp %>
+  <%= if error.data do %>
+    Data:
```

Comment:
This line is being displayed even when `custom_data` is configured as `:nothing`, so it changes the current behavior. Do we want that?
Same happens in the html template